src/ardrive.ts

```diff
@@ -1299,13 +1352,39 @@ export class ArDrive extends ArDriveAnonymous {
 		return { totalPrice: String(totalPrice), totalFilePrice: String(totalFilePrice), communityWinstonTip };
 	}
 
-	async getPrivateFolder(folderId: FolderID, driveKey: DriveKey): Promise<ArFSPrivateFolder> {
-		const folderEntity = await this.arFsDao.getPrivateFolder(folderId, driveKey);
-		return folderEntity;
+	async assertOwnerAddress(owner: ArweaveAddress): Promise<void> {
+		if (`${owner}` !== `${await this.wallet.getAddress()}`) {
```

Comment:
Javascript/Typescript's object equality stuff gives me a headache, but can we just implement an equality function on ArweaveAddress and use it here? If not possible via a built-in language feature, then something manual like:
```
equalsAddress(other: ArweaveAddress) {
    return this.address === other.address;
}
```